tests/tcp-rst-unacked-stream-09/suricata.yaml

```diff
@@ -0,0 +1,560 @@
+%YAML 1.1
```

Comment:
Why do we need a so big suricata.yaml ?

Reply:
it's not so big I think, just app later and eve sections really

Reply:
Sure, but how is it different than the default suricata.yaml ?

Reply:
(no big deal anyways)